skymp5-server/cpp/server_guest_lib/PieScript.cpp

```diff
@@ -119,15 +255,60 @@ uint32_t PieScript::GetSlotItem(int weaponChance, int armoryChacne,
 {
   std::pair<LootboxItemType, Tier> typeAndTier = AcknowledgeTypeAndTier(
     weaponChance, armoryChacne, consumableChance, nothingChance);
-  int item = GenerateRandomNumber(
-    0, lootTable[typeAndTier.first][typeAndTier.second].size() - 1);
-  return lootTable[typeAndTier.first][typeAndTier.second][item];
+  if (typeAndTier.first != LootboxItemType::Nothing) {
+    int item = GenerateRandomNumber(
+      0, lootTable[typeAndTier.first][typeAndTier.second].size() - 1);
+    return lootTable[typeAndTier.first][typeAndTier.second].at(item);
+  }
+  return 0;
+}
+
+void PieScript::Play(MpActor& actor)
+{
+  uint32_t item1 = GetSlotItem(80, 10, 10, 0);
+  uint32_t item2 = GetSlotItem(10, 80, 10, 0);
+  uint32_t item3 = GetSlotItem(25, 25, 40, 10);
+  uint32_t item4 = GetSlotItem(0, 0, 100, 0);
+
+  if (item1) {
+    actor.AddItem(item1, 1);
+  }
+  if (item2) {
+    actor.AddItem(item2, 1);
+  }
+  if (item3) {
+    actor.AddItem(item3, 1);
+  }
+  if (item4) {
+    actor.AddItem(item4, 1);
+  }
+}
+
+void PieScript::AddStarterKitItems(MpActor& actor, StarterKitType type)
+{
+  for (auto item : starterKitsMap[type]) {
```

Comment:
`item` is number here?

Reply:
Yes, here `item` represents item's index in `std::vector`.